stock_request/models/res_company.py

```diff
@@ -12,3 +12,6 @@ class ResCompany(models.Model):
     stock_request_allow_virtual_loc = fields.Boolean(
         string="Allow Virtual locations on Stock Requests"
     )
+    stock_request_check_available_stock_first = fields.Boolean(
```

Comment:
Simplify variable name at maximum, as stock is already mentioned:

```suggestion
    stock_request_check_available_first = fields.Boolean(
```